docs/src/guides/frameworks/drupal9/configuration.md

```diff
@@ -0,0 +1,81 @@
+---
```

Comment:
Er, why is this called `configuration.md`?

Reply:
Oh, just didn't  rename from your file  name. It's just the elasticsearch guide as  an example.